server/command.go

```diff
@@ -278,6 +283,55 @@ func executeHelp(p *Plugin, c *plugin.Context, header *model.CommandArgs, args .
 	return responsef(helpText)
 }
 
+func executeImport(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
+	posts, err := p.API.GetPostsForChannel(header.ChannelId, 0, 1)
```

Comment:
`header` is a bit confusing name for the command args. Generally we use `args` or `commandArgs`.